src/styles/blocks/about-us.scss

```diff
@@ -0,0 +1,44 @@
+.about-us {
+  margin: 0 auto;
+  margin-bottom: 120px;
+  @include padding-inline;
+
+  @include onTablet {
+    margin-bottom: 148px;
+  }
+
+  @include onSmallDesktop {
+    padding-inline: 154px;
+  }
+
+  @include onDesign {
+    max-width: 754px;
+    padding-inline: 0;
+  }
+
+  &__title {
+    color: $very-dark-blue;
```

Comment:
variables for colors should following naming convention like `$color-very-dark-blue` refactor this for the rest color variables to start with word `$color-`